cloudtrail-check.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+# 
+# This script is here to automate (as much as possible) the tasks in
+# https://cloud.gov/docs/ops/maintenance-list/#review-aws-cloudtrail-events
+# 
+# authorized events will include:
+#   a user name like i-17deadbeef1234567
+#   a source IP address within AWS.
+#   an AWS access key starting with ASIA
+# 
+# XXX The source IP range is kind of janky.  It would be better if this
+#     pulled dynamically from AWS and also if we handled cidr notation
+#     properly.  But these two ranges seem to work for govcloud right now.
+
+START=$(date -v -15d +"%Y-%m-%dT%H:%M:%SZ")
+echo "pulling cloudtrail from a bit over 2 weeks ago ($START)"
+
+aws cloudtrail lookup-events --start-time "$START" --lookup-attributes AttributeKey=EventName,AttributeValue=ConsoleLogin > /tmp/consolefoo.$$
```

Comment:
Does this handle pagination?

Reply:
I didn't experience any pagination while running this, but that's definitely something that I will add.  Thanks!

Reply:
@timothy-spencer do we need to handle pagination here? I'm confused.